docs/configuration.md

```diff
@@ -1481,6 +1481,17 @@ Apart from these, the following properties are also available, and may be useful
     How many batches the Spark Streaming UI and status APIs remember before garbage collecting.
   </td>
 </tr>
+<tr>
+  <td><code>spark.streamingListeners</code></td>
```

Comment:
this should be under the `spark.streaming` namespace instead, so a better name would be `spark.streaming.listeners`.